packages/react/src/components/navigation/navigationDropdownLink/NavigationDropdownLink.module.scss

```diff
@@ -0,0 +1,112 @@
+@import "../navigation.common";
+@value x-small-down, small-down, small-only, medium-only, medium-up, large-only, large-up, x-large-only, x-large-up from "../../../styles/breakpoints.css";
+
+.dropdownLinkItem.dropdownLinkItem {
+  &:focus-within {
+    box-shadow: none;
+  }
+
+  // dropdown menu
+  > div {
+    // remove margins from children
+    * {
+      margin: 0;
+    }
+
+    // dropdown menu item
+    > * {
+      font-weight: normal;
+
+      // dropdown menu item icon
+      span:first-of-type:not(:only-of-type) {
+        align-self: flex-start;
+        margin: 0 var(--spacing-2-xs) 0 calc(var(--spacing-3-xs) * -1);
+      }
+    }
+  }
+}
+
+.dropdownLinkItem {
+  a {
+    box-sizing: border-box;
+    color: inherit;
+    font-weight: 500;
+    outline: none;
+    text-decoration: none;
+  }
+}
+
+@media small-down {
+  .dropdownLinkItem.dropdownLinkItem {
+    align-items: flex-start;
+    border-radius: 0;
+    flex-direction: row;
+    flex-wrap: wrap;
+    // border-top: 1px solid var(--color-black-10);
+    > a {
+      flex: 1 0;
+      // border-bottom: 1px solid var(--color-black-10);
+    }
+    > button {
+      border: 1px solid var(--color-black-10);
+    }
+
+    // dropdown toggle button
+    > button,
+    > a {
+      height: var(--spacing-xl);
+      padding: var(--spacing-2-xs);
+
+      // focus outline
+      &:focus {
+        box-shadow: 0 0 0 var(--header-focus-outline-width) var(--header-focus-outline-color);
+      }
+
+      > span {
+        margin: 0;
+      }
+
+      // align angle to the right
+      > *:last-child {
+        margin-left: auto;
+      }
+
+      // dropdown icon
+      svg:first-of-type:not(:only-of-type) {
+        margin: 0 var(--spacing-2-xs) 0 calc(var(--spacing-3-xs) * -1);
+      }
+    }
+
+    // dropdown menu
+    > div {
+      --menu-border-width: 0;
+      --menu-width: calc(100% - (var(--spacing-xl) + var(--header-focus-outline-width)));
+      --menu-item-background-color: transparent;
+      --menu-item-background-color-highlighted: transparent;
+      --menu-item-color: inherit;
+      --menu-item-color-highlighted: var(--menu-item-color);
+      align-self: stretch;
+      margin-left: var(--spacing-l);
+      max-width: unset;
+      position: relative;
+      width: 100%;
+      // override inline min-width and top styles applied by the MenuButton component
+      min-width: auto !important;
+      top: 0 !important;
+
+      > * {
+        padding-left: var(--spacing-2-xs);
+
+        &:focus {
+          box-shadow: 0 0 0 var(--header-focus-outline-width) var(--header-focus-outline-color);
+        }
+      }
+    }
+  }
+}
+
+@media medium-up {
+  .dropdownLinkItem.dropdownLinkItem > button {
+      align-self: stretch;
```

Comment:
I did not see any effect with this. Maybe I am missing something 🤔

Reply:
The difference isn't much, but i believe it's meaningful. This is the same for the dropdownItem, except it includes the label area also.
<img width="171" alt="Screenshot 2022-06-29 at 15 49 54" src="https://user-images.githubusercontent.com/5700863/176440380-10c68b69-c93a-47a4-a51b-fc31fe763057.png">
<img width="167" alt="Screenshot 2022-06-29 at 15 49 48" src="https://user-images.githubusercontent.com/5700863/176440393-fdb419fd-62ae-41d4-9a88-e7e4ef8ae0e4.png">